packages/trpc/server/routers/viewer/payments.tsx

```diff
@@ -110,9 +107,11 @@ export const paymentsRouter = router({
         throw new TRPCError({ code: "BAD_REQUEST", message: "Invalid payment credential" });
       }
 
-      const paymentApp = await appStore[paymentCredential?.app?.dirName as keyof typeof appStore];
+      const paymentApp = (await appStore[
+        paymentCredential?.app?.dirName as keyof typeof appStore
+      ]?.()) as PaymentApp | null;
```

Comment:
Why are we adding the `?.()`? This wasn't there before

Reply:
The App Store is an index of functions. We weren't really calling them. Here's another instance: https://github.com/calcom/cal.com/blob/main/packages/trpc/server/routers/loggedInViewer/integrations.handler.ts#L176

Reply:
Basically we we're "awaiting" the function and not the promise it was supposed to return.